HedgeModManager/Languages/ko-KR.xaml

```diff
@@ -82,6 +84,7 @@
     <system:String x:Key="DiffUIModified">변경됨: {0}</system:String>
     <system:String x:Key="DiffUIRemoved" >제거됨: {0}</system:String>
     <system:String x:Key="DiffUIRenamed" >이름 변경됨: {0}</system:String>
+    <system:String x:Key="DiffUIMoved"   >이동됨: {0}</system:String>

```

Comment:
There is a conflict here, can you confirm these changes are correct? Please see https://github.com/thesupersonic16/HedgeModManager/commit/b229b601080f4e0fb4c9a82fd6af44b6d4b36cd2 for more information.
```suggestion
    <system:String x:Key="DiffUIModified">변경됨</system:String>
    <system:String x:Key="DiffUIRemoved" >제거됨</system:String>
    <system:String x:Key="DiffUIRenamed" >이름 변경됨</system:String>
    <system:String x:Key="DiffUIMoved"   >이동됨</system:String>
```